websites/docs.temporal.io/package.json

```diff
@@ -66,4 +66,4 @@
     "snipsync": "1.7.0",
     "temporalio": "^1.1.0"
```

Comment:
I *think* we can remove these from this package.json as well - because Assembly and the root package.json utilize these - not the site itself anymore.